runner/src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -396,11 +396,15 @@ export class PageControllerBase {
       const startPage = this.model.def.startPage;
       const formData = this.getFormDataFromState(state, num - 1);
 
-      if (
+      const isStartPage = this.path === `${startPage}`;
+      const isInitialisedSession = !!state.callback;
```

Comment:
I'm confused as to why this is needed now and not before

Reply:
Unsure why this pr is showing this as the changes are in main already 

![image](https://user-images.githubusercontent.com/97108643/179497129-bf22a359-e4b4-4ba2-8297-0008415d823a.png)